app/controllers/reviews_controller.rb

```diff
@@ -34,6 +34,20 @@ def update
     end
   end
 
+  def destroy
+    @review = Review.find(params[:id])
+    @park = @review.park
+    @review.destroy
+
+    if @review.destroyed?
+      flash[:notice] = "You have deleted review successfully!"
+      redirect_to park_path(@park)
+    else
+      flash[:notice] = "You failed to delete review!"
+      render "parks/show"
+    end
+  end
+
   private
```

Comment:
Same note and strategy as above, regarding abstracting out `@review = Review.find(params[:id])`